packages/expo-clipboard/ios/UIImage+gif.swift

```diff
@@ -0,0 +1,58 @@
+import ImageIO
+import MobileCoreServices
+import UIKit
+
+extension UIImage {
```

Comment:
I still don't see why this needs to be an extension. Can you explain?

Reply:
There is no specific reason, I just thought it would be easier to extend the gif function to process the data and return an animated UIImage, if it breaks the guidelines I can just remove it and make it a standalone function to process the data into the UIimage.